YAXLibTests/SerializationTest.cs

```diff
@@ -2512,11 +2726,7 @@ public void PolymorphicCollectionWithNullElements()
         [Test]
         public void SerializingNullValues()
         {
-            var book = new Book();
-            book.Price = 10;
-            book.Title = null;
```

Comment:
lol useful :P

Reply:
Price can't be null at least :)
The rest of blanks went here, strange. Will fix.